stiles.css

```diff
@@ -1,176 +1,181 @@
-html, body {
-    height: 100%;
-    margin: 0px;
-  }
-  
-  body{
-    margin: 0 auto;
-  }
-  
-  *{
-    box-sizing: border-box;
-  }
-  /* projects  */
-  
-  .header{
-    background-color: red;
-    height: 40px;
-    position: sticky;
-    top: 0;
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    padding-right: 20px;
-  }
-  
-  /*header  */
-  .header-content{
-    display: flex;
-    flex-direction: row;
-    gap: 20px;  
-  }
-  
-  .header-content a{
-    color: white;
-    text-decoration: none;
-    font-size: 1.3rem;
-    display: block;
-    padding: 0.5rem;
-  }
-  
-  .header-content a:hover{
-    background-color: #45567d;
-  }
-  
-  /* hero  */
-  #welcome-section{
-    background-color: black;
-    color: white;
-    height: 100vh;
-    width: 100vw;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-    
-  }
-  
-  #welcome-section h1{
-    font-size: 2rem;
-  }
-  #welcome-section i{
-    font-weight: ;
-  }
-  
-  /* list of project  */
-  
-  #projects{
-    max-width: 100%px;
-    display: flex;
-    justify-content: center;
-    background-color: #45567d;
-    padding: 2rem 4rem;
-    text-align: center;
-  }
-  
-  .project-container h2{
-    margin-top: 0;
-    color: white;
-    text-decoration: underline;
-    padding: 1rem;
-  }
-  
-  .project-container{
-    max-width: 900px;  
-    display: flex;
-    flex-direction: column;
-  }
-  
-  /* grid  */
-  .list-projects{
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    gap: 10px;
-    padding: 10px;
-    flex-wrap: wrap;
-    max-width: 800px;
-  }
-  
-  .list-projects article{
-    min-width: 250px;
-    max-width: 250px;
-    flex: 1 1 250px;
-    height: 250px;  
-  }
-  .card{
-    display: flex;
-    flex-direction: column;
-    overflow: hidden;
-    background-color: #303841;
-    justify-content: space-evenly;
-    
-    color: white;
-  }
-  img{
-    object-fit: contain;
+html,
+body {
+  height: 100%;
+  margin: 0;
+}
```

Comment:
I really apreciatte all your effort and love you put in this project but unfortunally I have to say that we need to make some changes in your css file. 👍👍👍👍

Please separate each css selector by adding a empty line between them. so that we your cs file has a better readibility.👍👍👍

I hope you have a great day 👍👍👍👍 and submit when you are done